apps/load_balancer/load_balancer.c

```diff
@@ -86,7 +86,7 @@ static void ev_handler(struct ns_connection *nc, int ev, void *ev_data) {
       if (* (int *) ev_data != 0) {
         /* TODO(lsm): mark backend as defunct, try it later on */
         fprintf(stderr, "connect(%s) failed\n",
-                s_http_backends[(int) nc->user_data].host_port);
+                s_http_backends[*(int *) nc->user_data].host_port);
```

Comment:
nc->user_data is actually a scalar casted to a void*.  It's written as a long and read as an intended though.